### ISSUE TITLE
cannot open blackb0x --> constantly crashing

### ISSUE THREAD

**Issue by tokisuno** [NSSpiral/Blackb0x#150]
i unplugged my microusb so i could use the hdmi (misread the steps), and now if i open blackb0x it crashes as soon as it detects my apple tv (3,2). any chances of saving this jailbreak?

**Comment by PunisherWarZone** [comment]
Yes plug it back to the TV & factory reset the Apple TV.   Redo the jailbreak
